dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -372,12 +372,31 @@ control dash_ingress(
         }
     }
 
-    apply {
-
-        /* Send packet on same port it arrived (echo) by default */
+// #ifdef UNDERLAY
 #ifdef TARGET_BMV2_V1MODEL
```

Comment:
Add a comment stating these are emulating SAI fixed header tables for underlay.

Reply:
bmv2/underlay.p4(39): [--Werror=not-found] error: underlay_routing: declaration not found
        underlay_routing.apply();
        ^^^^^^^^^^^^^^^^

Reply:
This has been wrapped under the ifdef